tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "esnext",
+    "target": "es2018",
+    "strict": true,
+    "declaration": true,
+    "outDir": "dist",
+    "removeComments": true,
+    "preserveConstEnums": true,
+    "allowJs": true
+  },
+  "files": [
+    "src/textarea-autosize.js"
```

Comment:
Thanks for moving this to a `src/` directory. Many of our repositories don't have one because the code base started out as a single source file. But as modules are added projects we really need a `src` directory to organize them.

Maybe we could standardize on `src/index.js` as the entry point  as well. I often spend a few seconds stumbling through my editor looking for the main file while bouncing between repositories.

Reply:
> Maybe we could standardize on `src/index.js` as the entry point as well. I often spend a few seconds stumbling through my editor looking for the main file while bouncing between repositories.

Yeah I think that's fair. Having a `src/` directory seems to be a typescript convention that I really like.